.github/workflows/build.yml

```
@@ -2,11 +2,14 @@ name: Build UI and Go App
 
 on:
   push:
+    branches: [main]
     branches: [main]
```

actionlint (1.7.8)

key "branches" is duplicated in "push" section. previously defined at line:5,col:5

(syntax-check)